src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -231,6 +231,20 @@ const SettingsModalStackNavigator = createModalStackNavigator([
         },
         name: 'Settings_Display_Name',
     },
+    {
+        getComponent: () => {
+            const SettingsTimezoneInitialPage = require('../../../pages/settings/Profile/TimezoneInitialPage').default;
+            return SettingsTimezoneInitialPage;
+        },
+        name: 'Settings_Timezone_Init',
```

Comment:
Hmm, kind of NAB but maybe rename `SettingsTimezoneInitialPage` to `SettingsTimezonePage` or possibly `SettingsMainTimezonePage` (though I prefer the former).

This is mainly because we don't seem to use ` X Initial Page` anywhere else in our naming convention except for [InitialSettingsPage](https://github.com/Expensify/App/blob/main/src/pages/settings/InitialSettingsPage.js) which is both slightly different and poorly named, IMO. 

Either way, we typically avoid abbreviations, so if you don't opt to rename this page, this should be `Settings_Timezone_Initial`

Reply:
Renamed routing constants, but I left the page name as it is, because I see we also have `SettingsInitialPage` and `WorkspaceInitialPage`. If we really don't like this naming we could create another PR to change all of them for the sake of consistency.

Reply:
Oh, you're right - I missed that one. It's more of a NAB so `SettingsInitialPage` is ultimately fine as long as we're not abbreviating. Thanks for updating things.